docs/APIReference-AtomicBlockUtils.md

```diff
@@ -24,3 +24,21 @@ insertAtomicBlock: function(
   character: string
 ): EditorState
 ```
+
+### moveAtomicBlockBefore
+
+```
+moveAtomicBlockBefore: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
+
+### moveAtomicBlockAfter
+
+```
+moveAtomicBlockAfter: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
```

Comment:
Or we forget about the before and after functions and implement a fourth (optional) argument insertionDirection to the moveAtomicBlock function, which overrides the algorithm which decides if the atomic block should be inserted before or after the targetRange.

```
type DraftInsertionDirection = 'before' | 'after';

moveAtomicBlock: function(
  editorState: EditorState,
  atomicBlock: ContentBlock,
  targetRange: SelectionState,
  insertionDireciton?: DraftInsertionDirection
): EditorState
```